index.html

```diff
@@ -84,6 +84,263 @@ <h1 class="primary-headline-text">Hey there. I’m Ansderly. <br>
         </div>
     </section>
 
+    <section class="recent-work-container">
+        <div class="recent-work-title-container">
+            <div class="recent-work-title">
+                <h1 class="text-recent-work">My Recent Works</h1>
+                <div class="line-work-container">
+                    <div class="line-work"></div>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="single-card-container">
+            <div class="single-card">
+                <div class="img-single-work">
+                </div>
+                <div class="infos-container">
+                    <div class="name-single-work">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                    </div>
```

Comment:
- [Optional] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).